source/_integrations/history_stats.markdown

```diff
@@ -35,6 +35,16 @@ sensor:
     type: time
     start: '{{ now().replace(hour=0, minute=0, second=0) }}'
     end: '{{ now() }}'
+    
+  - platform: history_stats
+    name: TV YouTube
+    entity_id: sensor.tv_source
+    state:
+      - "YouTube"
+      - "YouTube Kids"
+    type: time
+    start: '{{ now().replace(hour=0, minute=0, second=0) }}'
+    end: '{{ now() }}'
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.